helix-rest/src/main/java/org/apache/helix/rest/server/resources/helix/InstancesAccessor.java

```diff
@@ -222,11 +220,6 @@ private Response getParallelStoppableInstances(String clusterId, JsonNode node)
       _logger
           .error(String.format("Current cluster %s has issue with health checks!", clusterId), e);
       throw new HelixHealthException(e);
-    } catch (Exception e) {
```

Comment:
I think it is better to keep capturing the default Exception logging.

Reply:
I will leave the exception clean up work later. I found the catch statement here only catches a generic exception, first I think it's unlikely to occur and usually, it's not a good idea to catch all exceptions. The developer needs to know what's the right checked exceptions to catch, exceptions like NPE, IllegalException are "unchecked" exceptions for a reason.